### PR TITLE
Create _discord.okinea.json

### PR DESCRIPTION
**Pull request by okineadev**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

image

### CODE REVIEW

File: domains/_discord.okinea.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "okineadev",
+        "email": "81070564+okineadev@users.noreply.github.com"
```
Comment (CuteDog5695) [inline domains/_discord.okinea.json:4]: Please add another form of contact (like Discord, Twitter, Mastodon, Reddit etc.)

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/_discord.okinea.json (hunk 1 above)
Comment (okineadev) [reply domains/_discord.okinea.json:4]: Can I add a telegram?

Comment (DEV-DIBSTER) [reply domains/_discord.okinea.json:4]: > Can I add a telegram?

Yes.

Comment (okineadev) [reply domains/_discord.okinea.json]: Done!

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Pull request merged**